src/components/attendee/Input/Input.tsx

```diff
@@ -2,50 +2,47 @@ import React from "react";
 import { FieldError, RegisterOptions, UseFormRegister } from "react-hook-form";
 import classNames from "classnames";
 
-import { AnyForm, ContainerClassName } from "types/utility";
+import { AnyForm } from "types/utility";
 
 import CN from "./Input.module.scss";
 
-interface InputProps
-  extends React.HTMLProps<HTMLInputElement>,
-    ContainerClassName {
-  inputClassName?: string;
+type InputProps = React.HTMLProps<HTMLInputElement> & {
   error?: FieldError;
   onLabelClick?: () => void;
   label?: string;
   name?: string;
   register?: UseFormRegister<AnyForm> | (() => void);
   rules?: RegisterOptions;
   border?: "borderless" | "border";
-}
+  variant?: "login" | "overlay" | "overlay-profile" | "overlay-search"; // @debt: there should be a single "overlay" variant
+};
 
 export const Input: React.ForwardRefRenderFunction<
   HTMLInputElement,
   InputProps
 > = ({
-  containerClassName,
-  inputClassName,
   onLabelClick,
   error,
   label,
   register,
   rules,
   name,
   border = "borderless",
+  variant = "",
```

Comment:
[NOTE] I think this prop should have some default value or remove the optional parameter and make it required.

Reply:
Required might be a good idea. Will be better done in a new PR later on to uniform some more generic props (see the `<Spacer />` experimental component)